doc/attr.xml

```diff
@@ -1310,3 +1310,42 @@ fail]]></Example>
   </Description>
 </ManSection>
 <#/GAPDoc>
+  
+<#GAPDoc Label="HamiltonianPath">
+<ManSection>
+  <Prop Name="HamiltonianPath" Arg="digraph"/>
+  <Returns>A list or <K>fail</K>.</Returns>
+  <Description>
+    Returns a Hamiltonian path if one exists, <K>fail</K> if not.
+
+    A <E>Hamiltonian path</E> is a directed circuit visiting
+    each vertex exactly once. See Section <Ref Subsect="Definitions"
+     Style="Number" /> for the definition of a directed circuit.
+    Note the empty digraphs on 0 and 1 vertices are considered
+    Hamiltonian.<P/>
```

Comment:
You've not defined what Hamiltonian is so this doesn't make sense on its own. Instead, you could say something like `If <A>digraph</A> has 0 or 1 vertices, then <C>HamiltonianPath</C> returns [] or [1], respectively.`